projects/manage-supervision-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/compliance/Nsi.kt

```diff
@@ -0,0 +1,74 @@
+package uk.gov.justice.digital.hmpps.integrations.delius.compliance
+
+import jakarta.persistence.*
+import org.hibernate.annotations.Immutable
+import org.hibernate.annotations.SQLRestriction
+import org.springframework.data.jpa.repository.JpaRepository
+import java.time.LocalDate
+
+@Immutable
+@Entity
+@Table(name = "nsi")
+@SQLRestriction("soft_deleted = 0")
+class Nsi(
+
+    @Column(name = "offender_id")
+    val personId: Long,
+
+    @ManyToOne
+    @JoinColumn(name = "nsi_type_id")
+    val type: NsiType,
+
+    @Column(name = "event_id")
+    val eventId: Long?,
+
+    @Column(name = "actual_start_date")
+    val actualStartDate: LocalDate?,
+
+    @Column(name = "expected_start_date")
+    val expectedStartDate: LocalDate?,
+
+    @ManyToOne
+    @JoinColumn(name = "nsi_status_id")
+    val nsiStatus: NsiStatus? = null,
+
+    @Id
+    @Column(name = "nsi_id")
+    val id: Long = 0,
+
+    @Column(name = "active_flag", columnDefinition = "number")
+    var active: Boolean = true,
```

Comment:
Is this intended to be mutable?
```suggestion
    val active: Boolean = true,
```